apps/whale-api/src/module.api/rawtx.controller.ts

```diff
@@ -94,6 +95,20 @@ export class RawtxController {
     }
   }
 
+  @Get('/:txid')
+  async get (@Param('txid') txid: string, @Query('verbose', ParseBoolPipe) verbose: boolean = false): Promise<string | RawTransaction> {
+    try {
+      const rawTx = await this.client.rawtx.getRawTransaction(txid, verbose)
+      return rawTx
+    } catch (err) {
+      if ((err as RpcApiError)?.payload?.message === 'No such mempool or blockchain transaction. Use gettransaction for wallet transactions.') {
+        throw new BadRequestApiException('Transaction could not be found')
```

Comment:
```suggestion
        throw new NotFoundException('Transaction could not be found')
```

The test is failing because of this, this error should be NotFoundException?